projects/laji/src/app/shared/service/collection.service.ts

```diff
@@ -61,4 +97,35 @@ export class CollectionService extends AbstractCachedHttpService<ICollectionRang
       switchMap(cols => hasMore ? this.fetchWarehouseCollections(page + 1, cols) : of(cols))
     );
   }
+
+  getCollectionsTree(): Observable<any> {
```

Comment:
Would be good to have this typed (since it's a method in a service that's used else where) :)